test/graceful-shutdown-test.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const test = require('tape');
+const sinon = require('sinon');
+const schedule = require('..');
+
+test('Graceful Shutdown', function (t) {
+  let clock, sandbox, exitStub;
+  t.test('Setup', function (test) {
+    clock = sinon.useFakeTimers();
+    sandbox = sinon.createSandbox({ useFakeTimers: true });
+    exitStub = sandbox.stub(process, 'exit');
+    schedule.gracefulShutdown();
+    test.end();
+  });
+
+  t.test('close immediately when no job', function (test) {
+    schedule.gracefulShutdown().then(function() {
```

Comment:
what is the effect if `gracefulShutdown` is an empty method? Would it hang?

Reply:
No. Promise will be returned, but there will be no delay.

Reply:
Is this test actually testing anything then? Should we at least assert amount of active jobs at its end then?

Reply:
It doesn't seem to make any sense after removing the SIGINT callback. I will delete that test case.